### DISCUSSION TITLE
Did sqlalchemy get dropped from the release?

### DISCUSSION THREAD

**Discussion by narquette** [aws/aws-sdk-pandas#753] [General]
I see a method in catalog (awswrangler.catalog.get_enginer()) that can be used to generate a sqlalchemy engine.
Did this method get depricated?

**Comment by kukushking** [comment]
Hi @narquette,  you can create engine by means of sqlalchemy itself, see [docs](https://docs.sqlalchemy.org/en/14/core/engines.html). 

Wrangler doesn't not have an integration with sqlalchemy, but if there is a case for it we will certainly consider it.
